webapp/package.json

```diff
@@ -14,59 +14,59 @@
     "check-types": "tsc"
   },
   "devDependencies": {
-    "@babel/cli": "7.21.5",
-    "@babel/core": "7.21.8",
-    "@babel/preset-env": "7.21.5",
-    "@babel/preset-react": "7.18.6",
-    "@babel/preset-typescript": "7.21.5",
-    "@babel/runtime": "7.21.5",
+    "@babel/cli": "7.25.6",
+    "@babel/core": "7.25.2",
+    "@babel/preset-env": "7.25.4",
+    "@babel/preset-react": "7.24.7",
+    "@babel/preset-typescript": "7.24.7",
+    "@babel/runtime": "7.25.6",
     "@formatjs/cli": "6.2.12",
     "@mattermost/types": "7.10.0",
-    "@types/babel__core": "7.20.0",
-    "@types/babel__template": "7.4.1",
+    "@types/babel__core": "7.20.5",
+    "@types/babel__template": "7.4.4",
     "@types/node": "17.0.12",
     "@types/react": "16.14.26",
     "@types/react-bootstrap": "0.32.26",
     "@types/react-dom": "17.0.11",
     "@types/react-router-dom": "5.3.3",
-    "@types/react-transition-group": "4.4.6",
+    "@types/react-transition-group": "4.4.11",
     "@types/styled-components": "5.1.26",
-    "@typescript-eslint/eslint-plugin": "5.59.6",
-    "@typescript-eslint/parser": "5.59.6",
-    "@typescript-eslint/typescript-estree": "5.59.6",
+    "@typescript-eslint/eslint-plugin": "7.18.0",
+    "@typescript-eslint/parser": "7.18.0",
+    "@typescript-eslint/typescript-estree": "7.18.0",
     "babel-eslint": "10.1.0",
-    "babel-loader": "9.1.2",
-    "babel-plugin-formatjs": "10.5.1",
+    "babel-loader": "9.1.3",
+    "babel-plugin-formatjs": "10.5.16",
     "babel-plugin-styled-components": "2.1.1",
-    "eslint": "8.40.0",
+    "eslint": "8.57.0",
     "eslint-import-resolver-alias": "1.1.2",
-    "eslint-import-resolver-typescript": "2.7.1",
-    "eslint-import-resolver-webpack": "0.13.2",
+    "eslint-import-resolver-typescript": "3.6.3",
+    "eslint-import-resolver-webpack": "0.13.9",
     "eslint-plugin-babel": "5.3.1",
-    "eslint-plugin-cypress": "2.13.3",
+    "eslint-plugin-cypress": "3.5.0",
     "eslint-plugin-header": "3.1.1",
-    "eslint-plugin-import": "2.27.5",
-    "eslint-plugin-jest": "26.5.3",
-    "eslint-plugin-mattermost": "github:mattermost/eslint-plugin-mattermost",
-    "eslint-plugin-no-only-tests": "2.6.0",
-    "eslint-plugin-react": "7.32.2",
-    "eslint-plugin-react-hooks": "4.6.0",
+    "eslint-plugin-import": "2.30.0",
+    "eslint-plugin-jest": "28.8.3",
+    "eslint-plugin-no-only-tests": "3.3.0",
+    "eslint-plugin-react": "7.35.2",
+    "eslint-plugin-react-hooks": "4.6.2",
     "identity-obj-proxy": "3.0.0",
-    "webpack": "5.82.1",
-    "webpack-cli": "5.1.1"
+    "webpack": "5.94.0",
+    "webpack-cli": "5.1.4"
   },
   "dependencies": {
     "@floating-ui/react-dom-interactions": "0.13.3",
     "@mattermost/client": "7.10.0",
     "@mattermost/compass-icons": "0.1.38",
-    "core-js": "3.30.2",
-    "luxon": "3.3.0",
+    "@mattermost/eslint-plugin": "1.1.0-0",
```

Comment:
:tada:

Although I can't remember why we never published a non pre-release version of the package